### PR TITLE
[Feature: #164047234]: Add image to article

### PR DESCRIPTION
**Pull request by rajeman**
#### What does this PR do?
- It ensures images can be added to articles

#### Description of Task to be completed?
- ensure articles are created with image cover
- ensure image can be embedded within articles
- write unit tests
#### How should this be manually tested?
- goto `/create-article` to create an article

#### What are the relevant pivotal tracker stories?
- [#164047234](https://www.pivotaltracker.com/story/show/164047234)

#### Screenshots (if appropriate)
screen shot 2019-02-20 at 9 49 07 am

### CODE REVIEW

File: src/app/article/article.test.js [hunk 1]
```diff
@@ -161,6 +195,54 @@ describe('ARTICLE TEST SUITE', () => {
...
+        .find('Editor')
+        .props()
+        .init.images_upload_handler(blobInfo, success, failure);
+      expect(success).not.toHaveBeenCalled();
+    });
+
+    it('should call the file picker callback when file is selected', () => {
+      const component = shallow(<EditorComponent />);
+      const cb = jest.fn();
+      // console.log(component.find('Editor').props().init.file_picker_callback());
```
Comment (andela-mabdussalam) [inline src/app/article/article.test.js]: Commented code.

File: src/app/article/CreateArticleComponent.jsx [hunk 2]
```diff
@@ -12,30 +12,43 @@ class CreateArticleComponent extends React.Component {
     this.state = {
       body: '',
       canScrollToTop: false,
+      coverImageUrl:
+        'https://res.cloudinary.com/hba821/image/upload/v1550580103/iit15w9cb3jvbohazwke.png',
```
Comment (andela-mabdussalam) [inline src/app/article/CreateArticleComponent.jsx]: Is this the default cover image?

Comment (rajeman) [reply src/app/article/CreateArticleComponent.jsx]: No, It was. I should remove it.

**Comment by andela-mabdussalam** [comment]
Update typo in PR description

**Comment by rajeman** [comment]
I have done that

**Review by andela-mabdussalam — APPROVED** [review]

**Pull request merged**